example/Set-height-width-for-chatwidget.html

```
@@ -0,0 +1,48 @@
+<!doctype html>
+<html lang="en">
+   <head>
+      <title>Set height and width for Chat widget </title>
+   </head>
+   <body>
+      <h1> This sample code will help you to change the height and width of chat widget using CSS</h1>
+
+< script type = "text/javascript" >
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)